src/markdown_generators.js

```diff
@@ -59,13 +56,44 @@ const generateMarkdownRecursive = (page, courseData, pathLookup) => {
   const coursePageEmbeddedMedia = Object.values(
     courseData["course_embedded_media"]
   )
-    .map(embeddedMedia => {
-      embeddedMedia["course_id"] = courseData["short_url"]
-      embeddedMedia["type"] = "course"
-      embeddedMedia["layout"] = "video"
-      return embeddedMedia
+    .filter(
+      courseEmbeddedMedia => courseEmbeddedMedia["parent_uid"] === page["uid"]
+    )
+    .map(courseEmbeddedMedia => {
+      const embeddedMediaItems = courseEmbeddedMedia["embedded_media"].map(
+        embeddedMedia => {
+          let technicalLocation = embeddedMedia["technical_location"]
+          if (technicalLocation) {
+            const replacement = helpers.resolveRelativeLink(
+              technicalLocation,
+              courseData,
+              pathLookup,
+              true
+            )
+            if (replacement) {
+              technicalLocation = replacement
+            }
+          }
+
+          if (technicalLocation) {
+            return {
+              ...embeddedMedia,
+              technical_location: technicalLocation
```

Comment:
Can we run this through `stripS3`?  On prod `ocw-to-hugo` is run with this and the `staticPrefix` argument to redirect access to S3 resources through Fastly.

Reply:
@noisecapella might also want to add a test that spoofs running with `--stripS3` and `--staticPrefix=/coursemedia`

Reply:
I added a test, what do you think? We already run `stripS3` on `file_location` in `resolveRelativeLinks` and it doesn't look like there is a `technical_location` which has an AWS prefix. Should I still add `stripS3` here too?

Reply:
At the point `stripS3` is run in `resolveRelativeLinks`, isn't it still prefixed with `ocw.mit.edu`?  At this point, we've transformed it into an S3 link, and should run `stripS3` on it so that if the user has passed in these options, the link is modified as expected.  I'll take another look tomorrow to verify all this.

Reply:
`stripS3` is run on `file_location` in `resolveRelativeLinks`, and it doesn't look like there are any `file_location`s which start with a `ocw.mit.edu` prefix

Reply:
I see what you mean now, I was accidentally running with the wrong argument.